examples/dit/README.md

```diff
@@ -144,6 +156,29 @@ to launch a 4P training. For detail usage of the training script, please run
 bash scripts/run_distributed.sh -h
 ```
 
+## Evaluation
+
+### Inference
+
```

Comment:
Move the inference speed table to the samping section. I suggest to record the sampling method and the number of sampling steps in the table.